src/main/java/org/jabref/gui/IconTheme.java

```diff
@@ -32,24 +32,21 @@
 
 public class IconTheme {
 
-    public static Font FONT;
-    public static Font FONT_16;
-    public static javafx.scene.text.Font FX_FONT;
-
     /* Colors */
     // JabRef's default colors
     public static final Color DEFAULT_COLOR = JabRefPreferences.getInstance().getColor(JabRefPreferences.ICON_ENABLED_COLOR);
     public static final Color DEFAULT_DISABLED_COLOR = JabRefPreferences.getInstance().getColor(JabRefPreferences.ICON_DISABLED_COLOR);
+    private static final String DEFAULT_ICON_PATH = "/images/external/red.png";
+    private static final Log LOGGER = LogFactory.getLog(IconTheme.class);
```

Comment:
Please sort this and the following two variable to a different part of the class declaration. In the current form they are commented as "default colors".